mappers/lodging.js

```diff
@@ -70,7 +70,7 @@ export default function mapLodgings(records, translations) {
       if (type) {
         publicG.add(sym(lodgingUri), SCHEMA('keywords'), sym(type));
       } else if (type === undefined) {
-        console.error(`Cannot map location type value '${record['location_type']}' for record ${recordId}`);
+        errorLogger('location type', record['location_type'], recordId);
```

Comment:
Pass the same field name (`location_type`) as used to lookup the value as first argument. I know it was different before, but old code is wrong.